infection.json.dist

```diff
@@ -1,11 +1,11 @@
 {
     "source": {
         "directories": [
-            "test"
```

Comment:
Tests should not be infected - only the sources. This shortens execution of MTU from ~20 to ~1 minute (comparing to Travis job: https://travis-ci.org/php-slang/php-slang/jobs/376421082)